apps/mymedicare_cb/models.py

```diff
@@ -51,170 +47,246 @@ def get_and_update_user(subject, mbi_hash, hicn_hash, first_name, last_name, ema
+            err_msg = (
+                "UPDATE user_id_type as it has changed from the previous lookup value"
+            )
+            log_dict["status"] = "OK"
+            log_dict["mesg"] = err_msg
+            logger.info(log_dict)
             user.crosswalk.user_id_type = hash_lookup_type
             user.crosswalk.save()
 
         err_msg = "RETURN existing beneficiary record"
-        log_get_and_update_user(auth_flow_dict, "OK", user, fhir_id, mbi_hash, hicn_hash, hash_lookup_type, err_msg)
+        log_dict["status"] = "OK"
+        log_dict["mesg"] = err_msg
+        logger.info(log_dict)
         return user, "R"
     except User.DoesNotExist:
         pass
 
-    user = create_beneficiary_record(username=subject,
-                                     user_hicn_hash=hicn_hash,
-                                     user_mbi_hash=mbi_hash,
-                                     fhir_id=fhir_id,
-                                     first_name=first_name,
-                                     last_name=last_name,
-                                     email=email,
-                                     user_id_type=hash_lookup_type,
-                                     auth_flow_dict=auth_flow_dict)
+    user = create_beneficiary_record(
+        username=subject,
+        user_hicn_hash=hicn_hash,
+        user_mbi_hash=mbi_hash,
+        fhir_id=fhir_id,
+        first_name=first_name,
+        last_name=last_name,
+        email=email,
+        user_id_type=hash_lookup_type,
+        request=request,
+    )
 
     err_msg = "CREATE beneficiary record"
-    log_get_and_update_user(auth_flow_dict, "OK", user, fhir_id, mbi_hash, hicn_hash, hash_lookup_type, err_msg)
+    log_dict["status"] = "OK"
+    log_dict["mesg"] = err_msg
+    logger.info(log_dict)
     return user, "C"
 
 
 # TODO default empty strings to null, requires non-null constraints to be fixed
-def create_beneficiary_record(username=None,
-                              user_hicn_hash=None,
-                              user_mbi_hash=None,
-                              fhir_id=None,
-                              first_name="",
-                              last_name="",
-                              email="",
-                              user_id_type="H",
-                              auth_flow_dict=None):
-
-    if auth_flow_dict is None:
-        # If None, set empty dictionary.
-        auth_flow_dict = {}
+def create_beneficiary_record(
+    username=None,
+    user_hicn_hash=None,
+    user_mbi_hash=None,
+    fhir_id=None,
+    first_name="",
+    last_name="",
+    email="",
+    user_id_type="H",
+    request=None,
+):
+
+    logger = BasicLogger()
+
+    if request is not None:
+        logger = request._logger
+
+    log_dict = {
+        "type": "mymedicare_cb:create_beneficiary_record",
+        "username": username,
+        "fhir_id": fhir_id,
+        "user_mbi_hash": user_mbi_hash,
+        "user_hicn_hash": user_hicn_hash,
+    }
```

Comment:
```suggestion
        "user_hicn_hash": user_hicn_hash,
        "crosswalk": {},
    }
```

@nbragdon This isn't in the original logging, but I think adding the crosswalk info to the log here would be a good addition?  If adding, some tests might need updated too.

Reply:
Actually I added that there because it breaks without it. With it, the tests are all run successfully. Without it, it can't seem to add the crosswalk at a later time.